impeller/renderer/backend/vulkan/capabilities_vk.cc

```diff
@@ -328,7 +328,7 @@ bool CapabilitiesVK::SupportsSSBO() const {
 
 // |Capabilities|
 bool CapabilitiesVK::SupportsBufferToTextureBlits() const {
-  return true;
+  return false;
```

Comment:
Consider an ifdef with `!FML_OS_ANDROID` and a comment explaining this is implemented and working on non-Android platforms.

Reply:
I'm not confident in that given that I can run the gallery on vulkan outside of android

Reply:
err can't